### PR TITLE
feat: Update Russian translations

### PR DESCRIPTION
**Pull request by IZOBRETATEL777**
## Description

Update all  uncompleted translations
## Related Issues / Pull Requests

Resolve Update Translations #664

## How Has This Been Tested?

Ubuntu 20.04 KDE
## Checklist

- [ ] If the key of a setting is changed, the `old` attribute is updated or it is resolved in SettingsUpdater.
- [ ] If there are changes of the text displayed in the UI, they are wrapped in `tr()` or `QCoreApplication::translate()`.
- [ ] If needed, I have opened a pull request or an issue to update the [documentation](https://github.com/cpeditor/cpeditor.github.io).
- [ ] If these changes are notable, they are documented in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md).

### CODE REVIEW

File: translations/ru_RU.ts [hunk 1]
```diff
@@ -2533,52 +2534,53 @@ A test case will be elided and read-only if it&apos;s too long.</source>
...
     <message>
         <source>YAPF Style</source>
-        <translation type="unfinished"></translation>
+        <translation>Стиль YAPF</translation>
     </message>
     <message>
         <source>The YAPF style options, which are usually saved in a .style.yapf or setup.conf configuration file.
 You can learn about it by running `yapf --style-help`.</source>
-        <translation type="unfinished"></translation>
+        <translation>Параметры стиля YAPF, которые обычно сохраняются в конфигурационный файл style.yapf or setup.conf.
```
Comment (ouuan) [inline translations/ru_RU.ts:2574]:
```suggestion
        <translation>Параметры стиля YAPF, которые обычно сохраняются в конфигурационный файл .style.yapf or setup.conf.
```

File: translations/ru_RU.ts [hunk 2]
```diff
@@ -2533,52 +2534,53 @@ A test case will be elided and read-only if it&apos;s too long.</source>
...
     </message>
     <message>
         <source>Format the code when saving it manually.</source>
-        <translation type="unfinished"></translation>
+        <translation>Форматировать код, когда он сохраняется вручную.</translation>
     </message>
     <message>
         <source>Format the code when auto-saving it.</source>
-        <translation type="unfinished"></translation>
+        <translation>Форматировать код, когда он автоматический сохраняется.</translation>
```
Comment (sadykhzadeh) [inline translations/ru_RU.ts:2583]: @IZOBRETATEL777 автоматически* вроде

File: translations/ru_RU.ts [hunk 3]
```diff
@@ -1514,11 +1514,12 @@ If it&apos;s partially checked, the global setting in Code Edit will be used.</s
     </message>
     <message>
         <source>YAPF</source>
```
Comment (IZOBRETATEL777) [inline translations/ru_RU.ts:1516]: Can we remove this word from translations.
YAPF is YAPF.

File: translations/ru_RU.ts [hunk 4]
```diff
@@ -1185,11 +1185,11 @@ Do you want to reload it?</source>
     </message>
     <message>
         <source>You need to install CF Tool to submit your code to Codeforces. If already installed, you can add it in the PATH environment variable or check your settings at %1.</source>
-        <translation type="unfinished"></translation>
+        <translation>Вам необходимо устанвоить CF Tool для отправки Вашего кода на Codeforeces. Если он уже установлен, вы можете добавить его в переменную среды PATH или проверить Ваши настройки в %1.</translation>
```
Comment (ouuan) [inline translations/ru_RU.ts:1188]:
```suggestion
        <translation>Вам необходимо устанвоить CF Tool для отправки Вашего кода на Codeforces. Если он уже установлен, вы можете добавить его в переменную среды PATH или проверить Ваши настройки в %1.</translation>
```

File: translations/ru_RU.ts (hunk 3 above)
Comment (ouuan) [reply translations/ru_RU.ts:1516]: You can simply leave it blank and mark it finished.

File: translations/ru_RU.ts [hunk 5]
```diff
@@ -2533,52 +2534,53 @@ A test case will be elided and read-only if it&apos;s too long.</source>
     </message>
     <message>
         <source>Clang Format Program</source>
-        <translation type="unfinished"></translation>
+        <translation>Программа Clan Format</translation>
```
Comment (ouuan) [inline translations/ru_RU.ts:2537]:
```suggestion
        <translation>Программа Clang Format</translation>
```

**Review by sadykhzadeh — APPROVED** [review]
LGTM!

**Review by coder3101 — APPROVED** [review]

**Pull request merged**